### PR TITLE
sccp-51-rebalancing-sdefi

### PR DESCRIPTION
**Pull request by jacko125**
- closed #775  as the changes were made off `debt-snapshot` branch

### CODE REVIEW

File: publish/deployed/mainnet/synths.json [hunk 1]
```diff
@@ -66,63 +66,63 @@
...
-				"units": 9.52,
-				"weight": 7.2
+				"units": 0.4,
+				"weight": 7.5
 			},
 			{
-				"asset": "LEND",
-				"units": 581.76,
-				"weight": 6.7
+				"asset": "AAVE",
```
Comment (justinjmoses) [inline publish/deployed/mainnet/synths.json:78]: Standing by for node operators to support AAVE.

File: publish/deployed/mainnet/synths.json [hunk 2]
```diff
@@ -66,63 +66,63 @@
...
 			{
-				"asset": "BNT",
-				"units": 61.2,
-				"weight": 3.4
+				"asset": "CRV",
+				"units": 259.62,
+				"weight": 5.0
+			},
+			{
+				"asset": "NXM",
```
Comment (justinjmoses) [inline publish/deployed/mainnet/synths.json:113]: Needs to be changed to `WNXM` - as the wrapped is the decentralized version

Comment (jacko125) [reply publish/deployed/mainnet/synths.json]: updated

**Comment by jacko125** [comment]
test-deployments are breaking as staging rinkeby deployments are not included in develop yet

**Review by zyzek — APPROVED** [review]

**Review by hav-noms — APPROVED** [review]

**Pull request merged**